### ISSUE TITLE
MacOS: AppleScript support for UTM to minimize UTM window

### ISSUE THREAD

**Issue by LostInCompilation** [utmapp/UTM#4089]
It looks like UTM for macOS does not have AppleScript support.
I'm opening UTM and a VM with a bash script (open://utm...) and want to minimize the VM window with bash. I thought of AppleScript (osascript) using this as a test:
```
tell application "UTM"
	set miniaturized of window 1 to true
end tell
```
This results in an error.

Is there any other way to minimize UTM (or start minimized)?
If not, could the "miniaturized" property for AppleScript be added to UTM for macOS?

**Comment by Deadlyelder** [comment]
The UTM doesn't seem to return any of the typical properties when asked by AppleScript. For instance, the script below:

```bash
tell application "UTM"
	properties of window 1
end tell
```
throws the error `error "UTM got an error: Can’t get every property of window 1."`. I guess the properties need to be added to the compile build in order to be accessible to AppleScript?

**Comment by LostInCompilation** [comment]
Yes it seems like no properties are available. They need to be added to the build.
I also tried to use the open command with the "-j" option (launch hidden), however this completely crashes UTM. I made a new issue report for this.

**Comment by ryandesign** [comment]
> I thought of AppleScript (osascript) using this as a test:
> 
> ```
> tell application "UTM"
> 	set miniaturized of window 1 to true
> end tell
> ```
> 
> This results in an error (no properties found for UTM).

In UTM 4.7.5, this works, miniaturizing the window into the Dock.

> For instance, the script below:
> 
> tell application "UTM"
> 	properties of window 1
> end tell
> throws the error `error "UTM got an error: Can’t get every property of window 1."`.

This succeeds as well, returning:

```
{zoomable:true, closeable:true, zoomed:false, class:window, index:1, visible:true, name:"UTM", miniaturizable:true, id:406, miniaturized:false, resizable:true, bounds:{40, 25, 840, 677}, document:missing value}
```

Maybe this issue can be closed?
